### PR TITLE
Extend documentation for require.context

### PR DESCRIPTION
**Pull request by just-boris**
`require.context`, accepts an extra forth argument, but this is not described in documentation.

Information is based on replies from this issue https://github.com/webpack/webpack/issues/7283

### CODE REVIEW

File: src/content/api/module-methods.md [hunk 1]
```diff
@@ -353,17 +353,26 @@ Aside from the module syntaxes described above, webpack also allows a few custom
...
 ```
 
+If `mode` is specified as "lazy", the underlying modules should be loaded asynchronously: 
+
+```javascript
+var context = require.context('locales', true, /\.json$/);
+var localeA = await context('localeA');
+```
+
+The full list of available modes and its behavior is described in [`import()`](#import-) documentation.
```
[the remark this answers is not in the document]
Comment (just-boris) [reply src/content/api/module-methods.md]: On the current website this link points to the `import()`, exactly as I expected:

https://webpack.js.org/api/module-methods/#import-

[the remark this answers is not in the document]
Comment (montogeek) [reply src/content/api/module-methods.md]: I see, it differs from GH render, thanks for checking

**Comment by just-boris** [comment]
Added the missing argument, plus fixed an Eslint warning in the code snippet. Should be fine now.

File: src/content/api/module-methods.md [hunk 2]
```diff
@@ -353,17 +353,28 @@ Aside from the module syntaxes described above, webpack also allows a few custom
...
   directory: String,
   includeSubdirs: Boolean /* optional, default true */,
-  filter: RegExp /* optional */
+  filter: RegExp /* optional, default /^\.\/.*$/, any file */,
+  mode: String  /* optional, 'sync' | 'eager' | 'weak' | 'lazy' | 'lazy-once', default 'sync' */
 )
 ```
 
-Specify a whole group of dependencies using a path to the `directory`, an option to `includeSubdirs`, and a `filter` for more fine grained control of the modules included. These can then be easily resolved later on:
+Specify a whole group of dependencies using a path to the `directory`, an option to `includeSubdirs`, with a `filter` for more fine grained control of the modules included, and a `mode` to define the way how loading will work. Underlying modules can then be easily resolved later on:
```
Comment (EugeneHlushko) [inline src/content/api/module-methods.md]: `and` -> `with` change doesnt sound good to me, maybe just leave it as:
a path .... , an option ..., a filter ... and a mode

Comment (just-boris) [reply src/content/api/module-methods.md]: Updated

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**